presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -276,7 +276,7 @@ private static ConnectorTableMetadata getTableMetadata(String schemaName, TpchTa
     {
         ImmutableList.Builder<ColumnMetadata> columns = ImmutableList.builder();
         for (TpchColumn<? extends TpchEntity> column : tpchTable.getColumns()) {
-            columns.add(new ColumnMetadata(columnNaming.getName(column), getPrestoType(column)));
+            columns.add(new ColumnMetadata(columnNaming.getName(column), getPrestoType(column), null, null, false, false, emptyMap()));
```

Comment:
actually, tpch supports non null columns, there are no null values in tpch data.

Reply:
The `false` indicates that the column is not nullable.